nimble/include/nimble/hci_common.h

```diff
@@ -1234,6 +1245,57 @@ struct hci_add_dev_to_resolving_list {
 /* External data structures */
 extern const uint8_t g_ble_hci_le_cmd_len[BLE_HCI_NUM_LE_CMDS];
 
+struct hci_connless_cte_params {
```

Comment:
in new code we want to use `__attribute__((packed));` for all hci structures - please use it here as well.

as a reference please check for
 https://github.com/apache/mynewt-nimble/pull/507

also this code could be guarded with single flag fro Constant Tone Extensions